tests/cxx/built_ins_with_mapping.imp

```diff
@@ -0,0 +1,9 @@
+# Header mappings for IWYU tests.
+# Note that some headers listed here don't actually exist; we just want
+# IWYU to suggest using them
+[
+  { symbol: ["__builtin_expect", "private", "\"tests/cxx/built_ins_with_mapping-d2.h\"", "public"] },
+  { symbol: ["__builtin_strlen", "private", "\"tests/cxx/built_ins_with_mapping-d3.h\"", "public"] },
+  { symbol: ["__builtin_strcmp", "private", "\"tests/cxx/built_ins_with_mapping.h\"", "public"] },
```

Comment:
There's all sorts of special rules for the associated header, so maybe the test would be stronger if it used a non-associated header for this mapping/test scenario?

I'm fully prepared to accept it doesn't make a difference, but I thought I'd ask.

Reply:
As mentioned above, I'm specifically using the associated header because there are potential issues there.